packages/ove-asset-manager/src/OVE.Service.AssetManager/wwwroot/js/site.js

```diff
@@ -0,0 +1 @@
+﻿// Write your site wide JavaScript code.

```

Comment:
Is this file needed?

Reply:
not currently but I see little value in its removal. for example auth code would go here.

Reply:
Alternatively, we could also add that file at that point, where we need it, :).